lms/static/sass/base/_layouts.scss

```diff
@@ -0,0 +1,104 @@
+// base layout styles to support early responsive lms
+// may be discarded later once sass breakpoints are wired in
+
+body.view-incourse {
+  background-color: $body-bg;
+
+  .window-wrap {
+    min-width: 760px;
+  }
+
+  header.global,
+  header.global.slim {
+    width: auto;
+
+    .nav-wrapper {
+      min-width: auto;
+      padding-right: 2%;
+      padding-left: 2%;
+    }
+  }
+
+  .wrapper-course-material,
+  .wrapper-preview-menu {
+    padding: 0;
+  }
+
+  .wrapper-course-material .course-material,
+  .wrapper-preview-menu .preview-menu  {
+    padding: 15px 2%;
+    width: auto;
+  }
+
+  .wrapper-course-material .course-material .course-tabs {
+    padding: 0;
+  }
+
+  .container {
+    max-width: none;
```

Comment:
Are these `max` and `min` values overrides? Because they are being overridden by the following `width` value.